bats/tests/helpers/utils.bash

```diff
@@ -26,6 +26,17 @@ assert_nothing() {
     true
 }
 
+jq_output() {
+    jq -r "$@" <<<"${output}"
+}
+
+get_setting() {
+    local json_key=$1
+    run rdctl api /settings
+    assert_success
+    jq_output "${json_key}"
```

Comment:
I don't really like the name `json_key`, and I think the function will be most flexible if you just pass through all arguments to `jq_output`:

```suggestion
    run rdctl api /settings
    assert_success
    jq_output "$@"
```